client/lib/screen-title/index.js

```diff
@@ -13,16 +13,19 @@ import { setTitle as legacySetTitle } from './actions';
 export function subscribeToStore( store ) {
 	let title = store.getState().page.title;
 	let count = store.getState().page.unreadCount;
+	let siteId = store.getState().ui.selectedSiteId;
 
 	store.subscribe( () => {
 		let nextTitle = store.getState().page.title;
 		let nextCount = store.getState().page.unreadCount;
+		let nextSiteId = store.getState().ui.selectedSiteId;
```

Comment:
A few notes here that I must have overlooked last time:
- We prefer to use `const` in all cases unless a variable value is reassigned. In the scope above, `let` is appropriate for `title`, `count`, and `siteId` because the values change over time.
- While I dug a bit and found that `getState` simply returns a reference to the current state object, I'm always a bit wary when I see multiple calls to a function like this. Might ease concerns to assign and re-use a variable, e.g. `const state = store.getState();`
- When available or obvious to implement, prefer using a selector. In this case, [`getSelectedSiteId( state )`](https://github.com/Automattic/wp-calypso/blob/9c31ea61ab0ac58ac45cb7e8579960bff30a67d5/client/state/ui/selectors.js#L23-L31). While very minimal in this case, selectors are preferable because they allows us to change the underlying implementation without needing to update numerous sections of the codebase.
- The repetition is a bit bothersome between here and the above scope. I could imagine some helper property for `getTitleOptions( state )`, but not sure if it'd be any better than what you have here.